src/lib/components/sidebar/SidebarGroup.svelte

```diff
@@ -1,13 +1,24 @@
 <script lang="ts">
+  import { browser } from "$app/environment";
+  import { onMount } from "svelte";
   import { ChevronDown16 } from "svelte-octicons";
 
-  let collapsed = false;
+  export let collapsed = false;
+  export let name: string = undefined;
+
+  $: key = `SidebarGroup:${name}`;
+
   function toggle(val: boolean) {
     collapsed = !val;
+    if (name && browser) {
+      localStorage.setItem(key, String(collapsed));
```

Comment:
See #619 for a useful `StorageManager` utility. It provides key prefixing and value normalization.